packages/react-magma-dom/src/components/Input/Input.stories.tsx

```diff
@@ -327,6 +327,83 @@ NumberInput.parameters = {
   controls: { exclude: ['type', 'iconPosition', 'labelWidth'] },
 };
 
+export const PhoneInput = args => {
+  const [inputVal, setInputVal] = React.useState('');
+  const [hasError, setHasError] = React.useState(false);
+  const phonePattern = '^[0-9]{3}-[0-9]{2}-[0-9]{3}$';
```

Comment:
Sorry to be US centric, but lets update this to match US phone number patterns: 
```suggestion
  const phonePattern = '^[0-9]{3}-[0-9]{3}-[0-9]{4}$';
```

Reply:
Good catch )